src/main/java/com/redhat/cloud/notifications/NotificationsGwApp.java

```diff
@@ -17,22 +17,29 @@
 package com.redhat.cloud.notifications;
 
 import io.quarkus.runtime.StartupEvent;
+
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+
+import org.eclipse.microprofile.config.ConfigProvider;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 import org.jboss.logging.Logger;
 
+import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.event.Observes;
 
 /**
  * @author hrupp
  */
+@ApplicationScoped
 public class NotificationsGwApp {
 
+    private static final String NOTIFICATIONS_URL_KEY = "quarkus.rest-client.rbac.url";
```

Comment:
rbac url looks odd here. Shouldn't that be `.notifications-backend.url` or such?